<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CubeMapTextureBrowser</class>
 <widget class="QDialog" name="CubeMapTextureBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cubemap Texture Browser</string>
  </property>
  <widget class="QLineEdit" name="textRootPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSelectRootPath">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>11</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open directory</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="QtIcons.qrc">
     <normaloff>:/QtIcons/openscene.png</normaloff>:/QtIcons/openscene.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCreateCube">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>11</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Create cubemap texture</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="QtIcons.qrc">
     <normaloff>:/QtIcons/newscene.png</normaloff>:/QtIcons/newscene.png</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="listTextures">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1000</width>
     <height>683</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>611</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>There are no cubemap textures here. Try to create a new one or switch to another directory. </string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonReload">
   <property name="geometry">
    <rect>
     <x>713</x>
     <y>11</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Reload current directory</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="QtIcons.qrc">
     <normaloff>:/QtIcons/refresh.png</normaloff>:/QtIcons/refresh.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="selectedItemsStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>1001</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRemove">
   <property name="geometry">
    <rect>
     <x>985</x>
     <y>11</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete selected cubemap textures</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="QtIcons.qrc">
     <normaloff>:/QtTools/Icons/close-10.png</normaloff>:/QtTools/Icons/close-10.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="loadingWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>751</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>textRootPath</zorder>
  <zorder>buttonSelectRootPath</zorder>
  <zorder>buttonCreateCube</zorder>
  <zorder>label</zorder>
  <zorder>listTextures</zorder>
  <zorder>buttonReload</zorder>
  <zorder>selectedItemsStatus</zorder>
  <zorder>buttonRemove</zorder>
  <zorder>loadingWidget</zorder>
 </widget>
 <resources>
  <include location="QtIcons.qrc"/>
 </resources>
 <connections/>
</ui>
